### PR TITLE
added docs for prices

### PR DESCRIPTION
**Pull request by jpychan**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -1131,6 +1131,210 @@ Deletes a line item of an order.
...
+estimate_cents                         | integer          | optional | estimate price in cents, limit 8 digits
+charge_unit                            | string           | required | unit name
+order_unit                             | string           | required | unit name
+orderable                              | boolean          | required | default: true
+primary                                | boolean          | required | default: false
+estimate_by_default                    | boolean          | required | default: false
+created_at                             | string           | required | -
+updated_at                             | string           | required | -
+priceable_id                           | uuid             | optional | ID of the associated item
+priceable_type                         | string           | optional | type of the associated item
```
Comment (rbao) [inline source/index.html.md]: I think `priceable_id` and `priceable_type` is not exposed through the API, those are database details, they should not be included.

**Comment by rbao** [comment]
Please add space between `##` and the actual title to make it easier to read

File: source/index.html.md [hunk 2]
```diff
@@ -1131,6 +1131,210 @@ Deletes a line item of an order.
 
 `DELETE /v1/order_line_items/{order_line_item_id}`
 
-# Price
+#Price
 
-Coming soon...
\ No newline at end of file
+##The Price Resource
```
Comment (rbao) [inline source/index.html.md]: Don't use title case, just `The price resource`, be consistent with existing docs

**Comment by jpychan** [comment]
I made the above changes.

**Pull request merged**